app/utils/caching.js

```diff
@@ -0,0 +1,39 @@
+
+// # caching
+var _ = require('underscore');
+var _str = require('underscore.string');
+var path = require('path');
+var helmet = require('helmet');
+
+_.mixin(_str.exports());
+
+exports = module.exports = function(IoC) {
+  return {
+    getCachingHeadersFromSettings: function(settings){
+      var maxAge = parseInt(settings.staticServer.maxAge);
+      var headers = [];
+
+      if(maxAge){
```

Comment:
Might be best to rewrite this as returning the headers if there's no maxAge, to confirm with the coding style and don't indent if not necessary.